src/bin/add/add.rs

```diff
@@ -566,7 +566,7 @@ fn exec(mut args: AddArgs) -> CargoResult<()> {
             unknown_features.sort();
 
             if !unknown_features.is_empty() {
-                shell_warn(&format!("Unrecognized features: {:?}", unknown_features))?;
+                shell_warn(&format!("unrecognized features: {:?}", unknown_features))?;
```

Comment:
Okay! I feel plenty comfortable rewriting my commit history. I intentionally included that change in that commit in case you changed your mind about cargo add, but I definitely agree that it'd be better to rewrite those changes to appear more coherent outside the context of this PR.

Reply:
I'd like to leave `src/bin/add` left along as much as possible.  If it was to be updated as part of another API changing, that is understandable but any improvements are a dead end.